### DISCUSSION TITLE
Uniform Color detection

### DISCUSSION THREAD

**Discussion by sebasjimenez9808** [roboflow/notebooks#113] [Q&A] [answered]
Hi! Your video is truly awsome!
I tried to look for the twich stream that you mention where you identified the colors of the uniform of the players but I couldn't find it. Do you have any references that may be helpful here to identify players from different teams?

Also, I wanted to ask how would you approach the problem of trasnforming this to a bird eye's view?

Thank you in advance for your comments :)

**Comment by SkalskiP** [comment, marked as answer]
Twitch store stream recordings only for one month. After that, they are auto-deleted. But I still have Kaggle notebook: https://www.kaggle.com/code/skalskip/data-frame-post-processing

And some example results. I hope it is going to be helpful. It is not perfect, but probably a good starting point for further analysis. 

https://user-images.githubusercontent.com/26109316/236330287-5ae7c377-25d6-4d73-bf85-ddb167159d73.mp4

https://user-images.githubusercontent.com/26109316/236330350-f0e719fc-29ad-4b91-bd8b-6a9de2aa7ab5.mp4

https://user-images.githubusercontent.com/26109316/236330142-541276bc-5516-448d-931f-03100f4e42d4.mp4

https://user-images.githubusercontent.com/26109316/236330227-f75cf679-ed64-4d29-acab-78d87a4b27e9.mp4

**Reply by sebasjimenez9808** [reply]
Thank you so much for the quixk reply, I think the kaggle link is not working, is it possible for you to check that please?

**Reply by SkalskiP** [reply]
Hi, @sebasjimenez9808! Yup, you are right notebook was private :/ I'm very sorry about that. I made it public. It works in chrome incognito mode. Should work for you too.

**Reply by sebasjimenez9808** [reply]
Thank you a lot Piotr! it works well now
